content/response_integrations/third_party/pager_duty/actions/SnoozeIncident.py

```diff
@@ -24,9 +24,15 @@ def main():
         siemplify.LOGGER.info("Starting Snoozing process")
         try:
             incident = pager_duty.snooze_incident(email_from, incident_id)
-            output_message = "Successfully Created Incident\n"
-            result_value = True
-            status = EXECUTION_STATE_COMPLETED
+            if incident:
+                output_message = "Successfully Snoozed Incident\n"
+                siemplify.result.add_result_json(incident)
+                result_value = True
+                status = EXECUTION_STATE_COMPLETED
+            else:
+                output_message = "Incident wasnt snoozed\n"
+                result_value = True
+                status = EXECUTION_STATE_COMPLETED
```

Comment:
Instead of if-else, especially when the values repeat, please use defaults:

```suggestion
            result_value = True
            status = EXECUTION_STATE_COMPLETED
            siemplify.result.add_result_json(incident)
            output_message = "Incident wasnt snoozed\n"
            if incident:
                output_message = "Successfully Snoozed Incident\n"

```

Shouldn't we always have a json result, even if empty?
Please always add the incidents so to make sure references in playbooks still works as expected (a placeholder would still be resolved as {} instead of a string with the placeholder's name.